spec/support/vcr_setup.rb

```diff
@@ -5,9 +5,17 @@
 VCR.configure do |config|
   config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
   config.hook_into :webmock
-  config.ignore_localhost = true
   config.configure_rspec_metadata!
-  config.ignore_localhost = true
+
+  # Chrome calls a lot of services and they trip us up.
+  config.ignore_hosts(
+    "localhost", "127.0.0.1", "0.0.0.0",
+    "accounts.google.com",
+    "android.clients.google.com",
+    "clients2.google.com",
+    "content-autofill.googleapis.com",
+    "optimizationguide-pa.googleapis.com",
+  )
```

Comment:
That is a lot! I wonder if we will have to update this list with a future chrome update.

I also wonder if we can disable some/all with Chrome settings.. but I can't figure out how we configure Chrome anyway.
I see that we do something with Angular test config [here](https://github.com/openfoodfoundation/openfoodnetwork/blob/df4cf4b76809fad0a7aa1f463578b22b44e4158d/config/ng-test.conf.js#L52-L58), but it doesn't seem relevant.
While looking I found a flag mentioned [here](https://stackoverflow.com/questions/67879646/what-chrome-flags-that-i-should-enable-disable-to-improve-puppeteer-performance) that might help with general performance.

So in summary, it's probably not worth investigating, and easier just to maintain this list :)